app/models/edition.rb

```diff
@@ -230,6 +230,20 @@ def web_url
     Plek.current.website_root + base_path
   end
 
+  # @FIXME this is only supposed to be in for temporarily changing of date fields
+  def new_dates_valid?
+    within_1_sec = -> (date_a, date_b) do
```

Comment:
This doesn't actually do what is implied by `within_1_sec`, although I don't know if this is an issue:

```
[1] pry(main)> a = DateTime.parse('2017-01-01 10:00:00.999')
=> Sun, 01 Jan 2017 10:00:00 +0000
[2] pry(main)> b = DateTime.parse('2017-01-01 10:00:01.000')
=> Sun, 01 Jan 2017 10:00:01 +0000
[3] pry(main)> a.to_i == b.to_i
=> false
```

When the two dates are clearly within 1 second of each other.

Reply:
@thomasleese you're right, I was just being lazy as I figured the odds of it slipping over a second would generally be low. But I should and will sort it out.

Reply:
Should be fixed now.